Library/Homebrew/extend/os/mac/keg_relocate.rb

```diff
@@ -131,9 +131,10 @@ def relocated_name_for(old_name, relocation)
     old_prefix, new_prefix = relocation.replacement_pair_for(:prefix)
     old_cellar, new_cellar = relocation.replacement_pair_for(:cellar)
 
-    if old_name.start_with? old_cellar
+    case old_name
+    when /^#{old_cellar}/
```

Comment:
What's the difference/preference here?

Reply:
Hmm, good question. I think I had it in my mind that it needed to be changed because `start_with?` couldn't be called on a regex but looking at it now, `old_{cellar,prefix}` is the regex not `old_name`, so I think I just got them confused. I'll change it back to the original and see if that works as expected.